docs/operation/downgrade.rst

```diff
@@ -1,9 +1,50 @@
 Downgrade Guide
 ***************
-This section describes the logical steps for downgrading **MetalK8s**.
+Downgrading a MetalK8s cluster is handled via utility scripts which are
+packaged with your current installation.
+This section describes a reliable downgrade procedure for **MetalK8s**
+including all the components that make up the stack.
 
-To downgrade your cluster you need to run the utility script shipped
-with the current installation providing it with the destination version:
+Supported Versions
+******************
+.. note::
+
+    MetalK8 supports downgrade **strictly** from one supported
+    minor version to another. For example:
+
+    - Downgrade from 2.1.x to 2.0.x
+    - Downgrade from 2.2.x to 2.1.x
+
+    Please refer to the release notes for more information.
+
+Downgrade Pre-requisites
+************************
+Prior to beginning the downgrade step listed below, make sure to complete the
+pre-requisites listed in :doc:`/operation/preparation`.
+
+To try out the feasibility of your downgrade demands, run the following
+command.
```

Comment:
Not sure I understand here.
Is it to test if the downgrade will succeed?
If so, change it to "You can test if your environment will successfully downgrade with the following command."